src/components/common/Dialogs/AwardAndSmiteDialogs/ManageReputationDialogForm/ManageReputationDialogForm.tsx

```diff
@@ -277,12 +302,15 @@ const ManageReputationDialogForm = ({
           />
         </DialogSection>
       )}
-      {/* {onlyForceAction && (
-        <NotEnoughReputation
-          appearance={{ marginTop: 'negative' }}
-          domainId={Number(domainId)}
-        />
-      )} */}
+
+      {canOnlyForceAction && (
+        <DialogSection>
+          <NotEnoughReputation
+            appearance={{ marginTop: 'negative' }}
+            domainId={Number(domainId)}
```

Comment:
Needs to be `motionDomainId` like above as we are validating if the domain where the motion will be created in has reputation, not if the _targeted_ domain has